### PR TITLE
Development

### PR DESCRIPTION
**Pull request by nikasulo**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,708 @@
...
+                        <h5>
+                            MOST POPULAR
+                        </h5>
+                        <a href="#"><h2>Watch Apple's trailer for its show about the Apollo 11 mission, For All Mankind</h2></a>
+                        <ul class="flex-row">
+                            <li><a href="#">Ivan Mehta</a></li>
+                            <li>1 day ago</li>
+                        </ul>
+                    </div>
+                    <div class="ad-latest flex-row">
```
Comment (mosaaleb) [inline index.html]: Side tip: One of Adblock extensions techniques to recognize ads and then block them is by checking class names. Any class that contains `advertisment, ads, ad, etc` will be blocked by adblock, although they are not really ads. So I suggest to change the class names of any sections uses any of these keywords.

**Review by mosaaleb — CHANGES_REQUESTED** [review]
## Amazing work @Oluwadamilareolusakin, @CornetS28 :clap:

### The UI looks very neat :100: . Your code base is well organized. The readme file is okay. I am sure you guys puts a lot of effort on it. Still there some minor requirments needs to be handled first before moving on. 

#### Navigation bar:
- [x] It should use fixed positioning.
- [x] (mobile version) should display the TNW logo at the leftmost part of it.
- [x] (medium screen) should display the TNW logo at the leftmost part of it and covers the two nav bars.
- [x] (Big screens) logo should be hidden.
- [x] (mobile version) should have search and hamburger (bars) icon on the rightmost of it.

#### More usage of semantic tags in the main content:
For Every section in the main content:
- [x] It should be wrapped around a pair of `<section>` tags
- [x] Each section inside these sections, should be wrapped around `<article>` tag.

#### General:
- [x] Let's add [stickler](https://github.com/microverseinc/linters-config/tree/master/css), it's now easier that ever to use. Just follow the link and instructions.
- Question: why you guys have two css files `index.css` & `style.css`? 

Please submit a new code review, once you handled these requirements.

Happy coding :computer:

File: index.html (hunk 1 above)
Comment (nikasulo) [reply index.html]: The ads have been removed, thank you for the information

**Comment by samgaco** [comment]
Hi @Oluwadamilareolusakin and @CornetS28 :hand: 

The project looks amazing and you've implemented the required changes perfectly. Congratulations :clap: 

**The project is approved** :tada:  You can merge and submit the project completion form. _Please :pray: , try to add a project description to your readme file so it looks more professional._

Regards! :raised_hands:

**Comment by nikasulo** [comment]
@samgaco thanks man!

**Pull request merged**